novuhq/website
# feat: add framework page layout

**Describe what changes this pull request brings**

Add Framework page

**Steps to check:**

1. Open [preview](https://deploy-preview-303--novu-website.netlify.app/framework/)
2. Make sure that everything looks and works as expected

**BEFORE MERGE:**

- [x] Add appropriate links to all sections
